objects/ui2/ui2Event.self

```diff
@@ -550,7 +550,7 @@ Feel free to inherit me and override the ones you can implement.
             | 
             combo nonmodifierKeyCap isArrow ifTrue: [^ handleSmallArrowPress: combo IfCannot: b].
 
-            combo nonmodifierKeyCap isPrintable ifTrue: [^combo nonmodifierKeyCap printString do: [|:c| insert_char: c]].
+            combo nonmodifierKeyCap printString size <= 6 ifTrue: [^combo nonmodifierKeyCap printString do: [|:c| insert_char: c]].
```

Comment:
Bleh. That doesn't work, because characters like `č` are sent as two one-byte events:

```
Received: � 196
Received: � 141
```